connect-examples/v2/node_gift-cards/README.md

```diff
@@ -1,309 +1,359 @@
+The sample application does not implement a login authentication mechanism that is suitable for production deployment. Instead, you are allowed to log in as any customer in the seller’s account under the credentials you provide. Please consider this when using login/logout-related pieces of the source code.
 
 ## Setup
 
-1. Ensure that you have npm installed with Node.js version v10 or later (run `npm -v` in your terminal). If not, follow the instructions for your OS: [https://www.npmjs.com/get-npm](https://www.npmjs.com/get-npm) 
+1. Ensure that you have Node.js version v10 or later (run `npm -v` in your terminal). If not, follow the instructions for your OS: [https://nodejs.org](https://nodejs.org)
+
+2. Set your credentials:
+
+   1. Create a `.env` file at the top of this directory by copying the contents of the `.env.example` file
+   2. In the file, replace the placeholder texts with actual values for:
+      - `ENVIRONMENT` should be set to `sandbox` or `production`
+      - `SQUARE_APPLICATION_ID` and `SQUARE_ACCESS_TOKEN` can be found under the _Credentials_ tab in your Square application
+      - `SQUARE_LOCATION_ID` can be found under the _Locations_ tabs in your Square application
+
+   Navigate to the [Developer Dashboard](https://developer.squareup.com/apps) to manage and retrieve credentials for Square applications. For more information on creating a Square application, see [Getting Started](https://developer.squareup.com/docs/get-started#step-2-create-an-application).
+
+   **Warning:** Remember to use your credentials only for testing the sample application. If you plan to make a version of this sample application available for your purposes, use the Square [OAuth API](https://developer.squareup.com/docs/oauth-api/overview) to safely manage access to Square accounts.
 
-2. Set your credentials: 
-    1. You need a `.env` file to provide credentials. Square provides a `.env.example` file. You should make a copy and name it `.env`.
-    2. In the file, set the following:
-        1. Environment to Sandbox (for testing).
-        2. Provide other credentials for the Sandbox environment. 
- 
-    For more information, see [Getting Started](https://developer.squareup.com/docs/get-started#step-2-create-an-application). The Create an Application section explains introductory credentials information. 
+   **Sandbox testing:** You may configure this application to run using either Square `sandbox` and `production` environments and credentials. For testing, the `sandbox` environment is recommended because you can use fake credit or debit cards to test payments. To learn more about testing in the sandbox environment, refer to [Test in the Sandbox](https://developer.squareup.com/docs/testing/sandbox).
```

Comment:
```suggestion
   **Sandbox testing:** You may configure this application to run using either Square `sandbox` or `production` environments and credentials. For testing, the `sandbox` environment is recommended because you can use fake credit or debit cards to test payments. To learn more about testing in the sandbox environment, refer to [Test in the Sandbox](https://developer.squareup.com/docs/testing/sandbox).
```